File: Php/poo/gymRegistration/library/mediumPlan.php
<?php

    class mediumPlan extends user {

    private float $cost = 85000.0; 

    public function __construct(string $names, string $lastnames, int $identification, float $weight, float $height) {
        parent::__construct($names, $lastnames, $identification, $weight, $height);
    }

    public function showServices(): void {
        echo "Plan Basico - Servicios:<br>";
        echo " Acceso General al Gym <br>";
        echo " Clases Guiadas (yoga, spinning, etc.)<br>";
    }

    public function getCost(): float {
        return $this->cost;
    }
}

File: Java/poo/gymRegistration/library/mediumPlan.java
package poo.gymRegistration.library;

    public class mediumPlan extends user{
        private final double cost = 85000.0; 

    public mediumPlan(String names, String lastnames, int identification, double weight, double height) {
        super(names, lastnames, identification, weight, height);
    }

    public void getShowServices(){
        System.out.println("Plan Basico - Servicios:");
        System.out.println( "Acceso General al Gym ");
        System.out.println("Clases Guiadas (yoga, spinning, etc.) ");
    }

    public double getCost(){
        return this.cost;
    }
}
